### PR TITLE
Simplify report closure

### PR DESCRIPTION
**Pull request by vishalpandeyvip**
Simplify report closure added for intacct api.

### CODE REVIEW

File: sql/scripts/016-set-ccc-expense-state.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+rollback;
+begin;
+
+update 
+  expense_group_settings 
+set 
+  ccc_expense_state = expense_state
```
Comment (labhvam5) [inline sql/scripts/016-set-ccc-expense-state.sql:7]: Add a new line at the end.

File: sql/scripts/017-fill-ccc-last-synced-at.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+rollback;
+begin;
+
+update 
+  workspaces 
+set 
+  ccc_last_synced_at = last_synced_at
+
```
Comment (labhvam5) [inline sql/scripts/017-fill-ccc-last-synced-at.sql:8]: Add a new line at the end also there might be some space pls remove that.

File: sql/scripts/016-set-ccc-expense-state.sql (hunk 1 above)
Comment (labhvam5) [inline sql/scripts/016-set-ccc-expense-state.sql:7]: add `;` at the end.

File: sql/scripts/017-fill-ccc-last-synced-at.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+rollback;
+begin;
+
+update 
+  workspaces 
+set 
+  ccc_last_synced_at = last_synced_at
```
Comment (labhvam5) [inline sql/scripts/017-fill-ccc-last-synced-at.sql:7]: Add `;` at the end.

File: apps/sage_intacct/migrations/0019_merge_20221213_0857.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.14 on 2022-12-13 08:57
```
Comment (NileshPant1999) [inline apps/sage_intacct/migrations/0019_merge_20221213_0857.py:1]: why is this here

**Review by ashwin1111 — APPROVED** [review]
